redhat/frr.spec.in

```diff
@@ -106,16 +106,16 @@ Requires:       ncurses json-c
 Requires(pre):  /sbin/install-info
 Requires(preun): /sbin/install-info
 Requires(post): /sbin/install-info
-BuildRequires:  texi2html texinfo autoconf patch libcap-devel groff 
+BuildRequires:  texi2html texinfo autoconf automake patch libcap-devel groff
```

Comment:
This should be fixed in latest update. Fixed a few more BuildRequires and Requires as well (needs gcc, but frr 2.0 doesn't need flex) and other changes. Thanks for review.